tests/unit/factories/canonical-data-factory-test.ts

```diff
@@ -0,0 +1,64 @@
+import { expect } from 'chai'
+
+import Factory from '../../../src/factories/canonical-data-factory';
+import { Feature } from '../../../src/factories/factory-types'
+import NamedNode from '../../../src/named-node'
+import Literal from '../../../src/literal'
+import DefaultGraph from '../../../src/default-graph'
+import Empty from '../../../src/empty'
+
+describe('data-factory', () => {
+  describe('DataFactory', () => {
+    describe('#supports', () => {
+      it('collections', () => { expect(Factory.supports[Feature.collections]).to.be.false() })
+      it('defaultGraphType', () => { expect(Factory.supports[Feature.defaultGraphType]).to.be.false() })
+      it('equalsMethod', () => { expect(Factory.supports[Feature.equalsMethod]).to.be.true() })
+      it('identity', () => { expect(Factory.supports[Feature.identity]).to.be.false() })
+      it('id', () => { expect(Factory.supports[Feature.id]).to.be.true() })
+      it('reversibleId', () => { expect(Factory.supports[Feature.reversibleId]).to.be.false() })
+    })
+
+    describe('equals', () => {});
```

Comment:
Is this a way of leaving a marker that it would be good if tests for these would be written as well? If so, perhaps add an explicit `// TODO`?

Reply:
Added tests